dependency_graph/README.md

```diff
@@ -0,0 +1,10 @@
+
+To generate the `dependancy_graph.svg` the executables `tred` and `dot` are required from the
```

Comment:
This is a new README.md, specifically inside the dependendency_graph sub-directory, next to the python file and shell script.    https://github.com/ethereum/webthree-umbrella/tree/develop/dependency_graph